A6-client-retries.md

```diff
@@ -106,9 +106,9 @@ gRPC's call deadline applies across all attempts for a given RPC. For example, i
 
 ##### Exponential Backoff
 
-The `initialBackoff`, `maxBackoff`, and `backoffMultiplier` parameters determine the randomized delay before retry attempts.
+The `initialBackoff`, `maxBackoff`, and `backoffMultiplier` parameters tune the exponential delay before retry attempts.
 
-The initial retry attempt will occur at `random(0, initialBackoff)`. In general, the `n`-th attempt will occur at `random(0, min(initialBackoff*backoffMultiplier**(n-1), maxBackoff))`.
+The initial retry attempt will occur after `initialBackoff * random(0.8, 1.2)`.  After that, the `n`-th attempt will occur after `min(initialBackoff*backoffMultiplier**(n-1), maxBackoff) * random(0.8, 1.2))`.
```

Comment:
This means the backoff can be both:

1. less than `initialBackoff`, and
1. more than `maxBackoff`

(1) is probably OK but (2) seems to subvert a reasonable user's assumption of what `maxBackoff` would mean.

Reply:
Eric commented on this above.  I think it's fine for the jitter to make it exceed max backoff.  Note that this is consistent with the [connection backoff spec](https://github.com/grpc/grpc/blob/master/doc/connection-backoff.md).

Reply:
Since we independently (oops) noticed the same thing, maybe we should explicitly have a note that says "Note that the backoff used may exceed the configured maxBackoff by up to 20 percent"?

Reply:
Good idea.  Done.